server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java

```diff
@@ -152,6 +152,7 @@ protected boolean doStart(JobContext context) throws Exception {
                     break;
                 }
                 log.warn("Sql task execution failed, will continue to execute next statement.", e);
+                taskContext.getExceptionListener().onException(e);
```

Comment:
When there are many SQL, if each one fails the alerts become overwhelming

Reply:
only the latest exception will be saved, if exception is fetched by TaskResult, exception will be reseted. So various exception may be recorded if Task throw different exception one by one

Reply:
get it